multilang.sh

```diff
@@ -109,6 +109,10 @@ done
 
 shift $((OPTIND - 1))
 
+if grep  -q -e "-[^0-9]" <<< "$@"; then
+  usage
+fi
+
```

Comment:
I don't understand what the purpose of this edit was, but it creates some problems, when I run `./multilang.sh upbit fetchMarketById BTC-ETC`, it prints the usage and exits

------------------

This happens because **-E** is present within the arguments

```bash
% grep -o -e "-[^0-9]" <<< "upbit fetchMarketById BTC-ETC"
-E
```